docs/_ldi-extensions/ldes-discoverer.md

```diff
@@ -0,0 +1,82 @@
+
+For simplicity, we recommend passing the config as arguments, like shown below:
+
+```shell
+docker run ghcr.io/informatievlaanderen/ldi-ldes-discoverer --discoverer.url="https://brugge-ldes.geomobility.eu/observations/by-time?year=2023&month=05&day=11"
+```
+
+> **NOTE**: when an url contains a `&` symbol, which will be picked up by the shell as an operator. In this example, if
+> the url was not wrapped in quotation marks, the shell will try to execute three different command, where the last two
+> will be `month=05` and `day=11`, which will fail of course.
+>
+> To resolve this, make sure the url is encapsulated in quotation marks.
+
+### Example output
+
+```turtle
+@prefix prov: <http://www.w3.org/ns/prov#> .
+@prefix rdf:  <http://www.w3.org/1999/02/22-rdf-syntax-ns#> .
+@prefix tree: <https://w3id.org/tree#> .
+
+<https://brugge-ldes.geomobility.eu/observations/by-time?year=2023&month=05&day=11&hour=05&pageNumber=3>
+        tree:relation  [ rdf:type   tree:Relation;
+                         tree:node  <https://brugge-ldes.geomobility.eu/observations/by-time?year=2023&month=05&day=11&hour=05&pageNumber=4>
+                       ] .
+
+<https://brugge-ldes.geomobility.eu/observations/by-time?year=2023&month=05&day=11&hour=05&pageNumber=2>
+        tree:relation  [ rdf:type   tree:Relation;
+                         tree:node  <https://brugge-ldes.geomobility.eu/observations/by-time?year=2023&month=05&day=11&hour=05&pageNumber=3>
+                       ] .
+
+<https://brugge-ldes.geomobility.eu/observations/by-time?year=2023&month=05&day=11>
+        tree:relation  [ rdf:type    tree:InBetweenRelation;
+                         tree:node   <https://brugge-ldes.geomobility.eu/observations/by-time?year=2023&month=05&day=11&hour=00>;
+                         tree:path   prov:generatedAtTime;
+                         tree:value  "2023-05-11T00"
+                       ];
+        tree:relation  [ rdf:type    tree:InBetweenRelation;
+                         tree:node   <https://brugge-ldes.geomobility.eu/observations/by-time?year=2023&month=05&day=11&hour=05>;
+                         tree:path   prov:generatedAtTime;
+                         tree:value  "2023-05-11T05"
+                       ];
+        [...]
+        tree:relation  [ rdf:type    tree:InBetweenRelation;
+                         tree:node   <https://brugge-ldes.geomobility.eu/observations/by-time?year=2023&month=05&day=11&hour=20>;
+                         tree:path   prov:generatedAtTime;
+                         tree:value  "2023-05-11T20"
+                       ] .
+
+[...]
+
+<https://brugge-ldes.geomobility.eu/observations/by-time?year=2023&month=05&day=11&hour=05>
+        tree:relation  [ rdf:type   tree:Relation;
+                         tree:node  <https://brugge-ldes.geomobility.eu/observations/by-time?year=2023&month=05&day=11&hour=05&pageNumber=1>
+                       ] .
+
```

Comment:
Although I understand the point of showing the output.. i'm not a fan of using the brugge space.. can we just show a generic output?